circuit/protocol/Cargo.toml

```diff
@@ -0,0 +1,74 @@
+[package]
+name = "t3rn-protocol"
+version = "1.0.0-alpha.0"
+authors = ["t3rn Ltd <team@t3rn.io>"]
+edition = "2018"
+homepage = "https://t3rn.io"
+license = "Apache 2.0"
+repository = "https://github.com/t3rn/t3rn/"
+description = "A crate that hosts a common definitions that are relevant for the all t3rn pallets."
```

Comment:
Maybe mention that this contains the core protocol logic as well, otherwise it seems like it serves the same purpose as t3rn-primitives.

Reply:
#a12cb36